packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.8.2
```

Comment:
I think we should strongly consider making this 0.9, given the scale of the changes here. I know it's not a breaking change from an API standpoint, but our previous test coverage was not great so with a rewrite of this magnitude there's a pretty good chance that there are going to be some regressions, even though we also know it will fix a lot of things. Making it a major version bump will require people to opt in to in, which means they'll be much more likely to understand that they should retest their critical cases on Android, look for feedback from their users, etc.